firebase.json

```diff
@@ -31,13 +32,19 @@
       "port": 5001
     },
     "firestore": {
-      "port": 8080
+      "port": 8000
     },
     "hosting": {
       "port": 5000
     },
     "ui": {
       "enabled": true
+    },
+    "database": {
+      "port": 9000
+    },
+    "pubsub": {
+      "port": 6000
```

Comment:
This is to enable scheduled cloud function triggering?

Reply:
yeah, scheduled functions are pubsub.